subfunctions/f_CaBBI_QGIF

```diff
@@ -0,0 +1,56 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% QGIF evolution function:
+%
+%                   [fx] = f_CaBBI_QGIF(Xt,Theta,Isyn,inF)
+%
+% this function contains the equations and parameters of the QGIF neuron
+% model, and the over-time integration scheme
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%  
+
+function [fx] = f_CaBBI_QGIF(Xt,Theta,Isyn,inF)
```

Comment:
Would it be difficul to also compute the derivatives df/dx and df/dtheta? Or performance is not an issue?